libs/core/langchain_core/runnables/config.py

```diff
@@ -291,7 +307,7 @@ def merge_configs(*configs: Optional[RunnableConfig]) -> RunnableConfig:
                     **(config.get(key) or {}),  # type: ignore
                 }
             elif key == "tags":
-                base[key] = list(  # type: ignore
+                base[key] = sorted(  # type: ignore
```

Comment:
ooc is this related to fix

Reply:
set has undefined order, so it was a mistake to pass things through set without sorting them after